src/main/java/kr/codesqaud/cafe/controller/UserLogicController.java

```diff
@@ -0,0 +1,48 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.basic.User;
+import kr.codesqaud.cafe.repository.UserMemoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/user")
+public class UserLogicController {
+    UserMemoryRepository userMemoryRepository;
```

Comment:
```suggestion
    UserRepository userMemoryRepository;
```

이름을 보니 요렇게 인터페이스를 썼어도 좋았을 것 같습니다.